.travis.yml

```diff
@@ -103,9 +103,9 @@ script:
   - if [ "$PHP_VERNUM" -ge "70300" ]; then phpenv config-rm xdebug.ini || true; fi
 
   - vendor/bin/codecept build --quiet
-  - vendor/bin/codecept run --ext DotReporter tests/cli/
-  - vendor/bin/codecept run --ext DotReporter tests/integration/
-  - vendor/bin/codecept run --ext DotReporter tests/unit/
```

Comment:
But why?

Reply:
That's just for this PR - to make it easier to find where the failed/errored tests are. I'll change it back before making this PR final.